level.go

```diff
@@ -0,0 +1,77 @@
+package logger
+
+import "go.uber.org/zap/zapcore"
+
+/*
+DEFAULT 	(0) The log entry has no assigned severity level.
+DEBUG 	(100) Debug or trace information.
+INFO 	(200) Routine information, such as ongoing status or performance.
+NOTICE 	(300) Normal but significant events, such as start up, shut down, or a configuration change.
+WARNING 	(400) Warning events might cause problems.
+ERROR 	(500) Error events are likely to cause problems.
+CRITICAL 	(600) Critical events cause more severe problems or outages.
+ALERT 	(700) A person must take an action immediately.
+EMERGENCY 	(800) One or more systems are unusable.
+*/
+
+type GoogleSeverity int
+
+func (l GoogleSeverity) String() string {
+	switch l {
+	case 0:
```

Comment:
I see this numbers are defined in two places, is it possible to create a constants\enum or reuse them somehow?

Reply:
Good point, I will re-use constants there

Reply:
Done